.github/workflows/ci.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.7']
+        python-version: ['3.7', '3.10']
```

Comment:
That double the long running test time. can we skip the 3.10 tests on everything but `unit-quick` ?